### PR TITLE
New Subcommand: List Collections

### PR DESCRIPTION
**Pull request by ghost**
https://github.com/open-contracting/kingfisher/issues/199

### CODE REVIEW

File: ocdskingfisher/collection_util.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                    if os.path.isdir(os.path.join(directory, version)) and \
+                            os.path.isfile(os.path.join(directory, version, "metadb.sqlite3")):
+                        _get_all_collections_disk_collection_found(
+                            source_id=key,
+                            data_version=version,
+                            sample=sample,
+                            out=out,
+                        )
+
+    # Return results
```
Comment (kindly) [inline ocdskingfisher/collection_util.py]: Would be nice if the results were sorted by source_id so that all collections from same source are together.

**Review by kindly — APPROVED** [review]
Mainly good only ordering would be nice.

File: ocdskingfisher/collection_util.py (hunk 1 above)
Comment (odscjames) [reply ocdskingfisher/collection_util.py]: My guess was that there would be different orderings people would want, so maybe we'd add a command line flag later. Tho maybe if we checked out the user cases only ordering by source would come up - I didn't give it too much thought, I admit.

**Comment by odscjames** [comment]
Thanks!

**Pull request merged**